LICENSES/vendor/github.com/hashicorp/errwrap/LICENSE

```diff
@@ -0,0 +1,358 @@
+= vendor/github.com/hashicorp/errwrap licensed under: =
+
+Mozilla Public License, version 2.0
```

Comment:
I don't think MPL is on the [CNCF license list][1] which we use per [vendor.md][2]

[1]: https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist
[2]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md#reviewing-and-approving-dependency-changes

Reply:
this appears to apply to all of the new hashicorp deps at a glance.

Reply:
It's gone.